src/master/flags.cpp

```diff
@@ -460,6 +460,18 @@ mesos::internal::master::Flags::Flags()
       "load an alternate allocator module using `--modules`.",
       DEFAULT_ALLOCATOR);
 
+  add(&Flags::hierarchical_recovery_factor,
+      "hierarchical_recovery_factor",
```

Comment:
`allocator_agent_recovery_factor`? (see my comment on the word  "hierarchical" below)